resources/messages/GrammarKitBundle.properties

```diff
@@ -14,6 +14,9 @@ action.grammar.Generate.JFlexLexer.text=Generate JFlex Lexer
 action.grammar.Generate.ParserUtil.text=Generate Parser Util
 action.grammar.Run.JFlex.text=Run JFlex Generator
 action.grammar.Generate.text=Generate Parser Code
+action.grammar.fleet.Generate.text=Generate Fleet-Compatible Parser Code
```

Comment:
IMHO: `Fleet: ....` name for actions would be quicker to scan

I'm concerned about "pollution" of the context menu, assuming that only few users will need Fleet-specific actions

Reply:
I suppose that we can hide Fleet-related actions behind a checkbox in Settings. But those who need Fleet-related would need to learn about this from the documentation.

Reply:
IMHO that would be overkill and too hard to discover.

Reply:
What if we show those actions if we detect fleet-related libraries referenced in the project?

Reply:
That would be the best solution probably.